packages/react-components/src/components/UnsupportedEnvironment.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { Icon, Link, Stack, Text } from '@fluentui/react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import {
+  containerStyles,
+  iconStyles,
+  linkTextStyles,
+  mainTextStyles,
+  secondaryTextStyles
+} from './styles/UnsupportedEnvironment.styles';
+import { UnsupportedBrowserStrings } from './UnsupportedBrowser';
+import { UnsupportedBrowserVersionStrings } from './UnsupportedBrowserVersion';
+
+/**
+ * props for UnsupportedBrowser UI
+ *
+ * @beta
+ */
+export interface UnsupportedEnvironmentProps {
+  /** Handler to perform a action when the help link is actioned */
+  onTroubleshootingClick?: () => void;
+  /** String overrides for the component */
+  strings: UnsupportedBrowserStrings | UnsupportedBrowserVersionStrings;
+}
+
+/* @conditional-compile-remove(unsupported-browser) */
+const UnsupportedEnvironmentContainer = (props: UnsupportedEnvironmentProps): JSX.Element => {
+  const { onTroubleshootingClick, strings } = props;
+  return (
+    <Stack styles={containerStyles}>
+      <Icon styles={iconStyles} iconName="UnsupportedBrowserWarning" data-ui-id="unsupportedBrowserIcon"></Icon>
+      <Text styles={mainTextStyles}>{strings.primaryText}</Text>
+      <Text styles={secondaryTextStyles}>{strings.secondaryText}</Text>
+      {onTroubleshootingClick && (
+        <Link
+          styles={linkTextStyles}
+          onClick={() => {
+            onTroubleshootingClick();
+          }}
+          data-ui-id="unsupportedBrowserLink"
+        >
+          {strings.moreHelpLink}
```

Comment:
This prop's name `moreHelpLink` suggests that this is link, but the `onTroubleshootingClick` handler name suggests (and in reality Contoso could use it to) that it is really just a button that takes _some_ action.

Consider renaming `moreHelpLink` to `moreHelpText` or even `moreHelpButtonText`.